src/mRides-server/Logic/UserCatalog.cs

```diff
@@ -144,18 +144,30 @@ public User create(User obj, int userId)
 
         public void setGSD(int userId, long amountGSD)
         {
-            //User u = _context.Users.Find(userId);
             User u = get(userId);
             u.GSD = amountGSD;
             _context.SaveChanges();
         }
 
         public long getGSD(int userId)
         {
-            //User u = _context.Users.Find(userId);
             User u = get(userId);
             return u.GSD;
         }
+
+        public string getGender(int userId)
+        {
+            User u = get(userId);
+            return u.gender;
+        }
+
+        public void setGender(int userId, string userGender)
+        {
```

Comment:
Yes you're right, but we don't have the method at this point and have another method in the server for modifying an attribute and these are the only ones we will need in the application, so I don't think it's necessary right now